pyproject.toml

```diff
@@ -1,13 +1,12 @@
-[tool.poetry]
+[project]
 name = "youtrack-sdk"
-version = "1.1+SNAPSHOT"
 description = "YouTrack SDK"
-authors = ["moneymeets <service@moneymeets.com>"]
+dynamic = [ "version" ]
+authors = [
+    { name = "moneymeets", email = "admin@moneymeets.com" },
+]
 readme = "README.md"
 repository = "https://github.com/moneymeets/youtrack-sdk"
-packages = [
-    { include = "youtrack_sdk" },
-]
```

Comment:
For public repositories that will be published to PyPI it's better to specify packages explicitly. I would keep this line.